roles/dosomething.base/files/mrich-authorized_keys

```diff
@@ -0,0 +1,2 @@
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCEi9YSXN4UT6ut1SpXL7bglWPxw4R9tbqtm5KCQRo7nE/RHC+T3ekl4RYLgiD4uqFHzYzoYfpLorxB6HqmkIydt3J/k1HfPjQoCanXM68rEnBG0BiJS7+VvKMzxxk6YmOJYhnf6mvKxmSvG0oySV5owRquyuY2asvXMW4HemzJKlk1cAK4RFIhC3GG8rEir61QPqVJG3T4qcBWA9q3jvoo7EvjVxXeuaQFBWvMZYctOIM98c8/ZAUO6G7eRh9LbrDsmuK3N3O6djsivVFoB6hway+M/kR8qOxk95woG+68AoF4rQHukJXXyTtufze3PVE6ym7iCDF5+4YXj0KoyM71 dosomething-mrich
```

Comment:
It'd be great to move keys out of role as well.
Could you please move keys to [files](https://github.com/DoSomething/tower/tree/master/files) folder? You can create a subfolder under it (f.i. `devops_users`), and change the lookup, for example:
```yml
    key="{{ lookup('file', 'files/devops_users/' + item + '-authorized_keys') }}"
```